tests/unit/test_config.py

```diff
@@ -181,7 +175,7 @@ def test_xfunction_attribute_error(self, mock_glbl_cfg, tmp_path):
         with pytest.raises(XtriggerConfigError) as excinfo:
             WorkflowConfig(workflow="capybara_workflow", fpath=flow_file,
                            options=SimpleNamespace())
-        assert "not found" in str(excinfo.value)
+        assert "module 'capybara' has no attribute 'capybara'" in str(excinfo.value)
```

Comment:
🐀.🐖

Reply:
I presume this is comment rather than critique 😁 ?

Reply:
I think that's mouse - crossproduct - pig